openlibrary/templates/merge_request_table/table_row.html

```diff
@@ -26,7 +26,7 @@
         $# End : Status dot indicator
 
         <div class="mr-details">
-        $:_('<span class="mr-details__request-title">%(title)s</span>', title=request_title)
+        $:_('<span class="mr-details__request-title"><a href="%(url)s" target="_blank">%(title)s</a></span>', title=request_title, url=request["url"])
```

Comment:
```suggestion
        <span class="mr-details__request-title">$request_title</span>
```
Remove link.  Render span itself --- there's nothing to translate, so no need to use `gettext` here.